core/tests/root-files-config.json

```diff
@@ -60,7 +60,10 @@
     "core/tests/ci-test-suite-configs/README.md",
     "core/templates/AppSpec.ts",
     "core/templates/domain/skill/MisconceptionObjectFactorySpec.ts",
-    "core/templates/domain/exploration/AnswerGroupObjectFactorySpec.ts"
+    "core/templates/domain/exploration/AnswerGroupObjectFactorySpec.ts",
+    "core/tests/puppeteer-acceptance-tests/specs/exploration-editor/manage-exploration-misconceptions.spec.ts",
```

Comment:
Why is this acceptance test being moved here?

Reply:
A test was failing that asked me to move it to `root-files-config.json`. I have removed it for now, lets see if it fails again.